### PR TITLE
port print_env.py to print_env.sh

### PR DESCRIPTION
**Pull request by geoffwa**
I'm not sure if this works in *actualy* bourne shell or whatever Alpine is using these days.

### CODE REVIEW

**Comment by geoffwa** [comment]
#incrementalwins

File: scripts/print_env.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# set -ex
+
+usage() {
+  echo 'Expected usage: `print_env $name`. Must be a string at least 1 character in length.' >&2
+  exit 1
+}
+
+[[ -n "$1" ]] || usage
+
+(set -o posix; set) | while read -r line; do
```
Comment (VSpike) [inline scripts/print_env.sh]: Wouldn't `env` be better than `(set -o posix; set)` here?

Comment (geoffwa) [reply scripts/print_env.sh]: `set` is a shell built-in, `env` isn't [1]

(repeat about built-ins ad nauseam)

**Comment by geoffwa** [comment]
After a chat with @VSpike I'm not sure how much of a win this is given we need Python for `awscli` and having an /actual/ programming language for dealing with shell/env vars is pretty useful, so I'm going to bin this. I think my main gripe is having to copy a docker image to a separate job in CircleCI in order to use the build tooling, but there's probably a better way to solve it than what I started here.

**Comment by marcinwyszynski** [comment]
> there's probably a better way to solve it

[Yes, there is](https://circleci.com/orbs/).

**Pull request closed without merging**